hypha/static_src/javascript/csrf-token-ajax.js

```diff
@@ -1,30 +1,4 @@
 (function ($) {
-    // reset mobile filters if they're open past the tablet breakpoint
-    $(window)
-        .resize(function resize() {
-            if ($(window).width() > 1024) {
-                $(".js-actions-toggle").removeClass("is-active");
-                $(".js-actions-sidebar").removeClass("is-visible");
-                $(".tr--parent.is-expanded").removeClass("is-expanded");
-            }
-        })
-        .trigger("resize");
-
-    $("form")
-        .filter(".form__comments")
-        .submit(function (e) {
-            var $form = $(this);
-            var formValues = $form.serialize();
-            var previousValues = $form.attr("data-django-form-submit-last");
-
-            if (previousValues === formValues) {
-                // Previously submitted - don't submit again
-                e.preventDefault();
-            } else {
-                $form.attr("data-django-form-submit-last", formValues);
-            }
-        });
-
     // Setting the CSRF token on AJAX requests.
```

Comment:
@theskumar I believe I added this when I created the flag feature and used fetch/ajax to set/unset the flags. That is now converted in to htmx, do we need it?

At https://django-htmx.readthedocs.io/en/latest/tips.html#make-htmx-pass-the-csrf-token it seems csrftoken is needed when doing `hx-post`. Maybe we can remove the script and add the `hx-headers` stuff instead whenever we use `hx-post`?